### DISCUSSION TITLE
Upgrade to PatternFly 5

### DISCUSSION THREAD

**Discussion by andreas-blaettlinger** [keycloak/keycloak#22344] [Ideas]
I've been looking to bring some quality of life improvement to the Admin UI. Just had the idea to introduce a password-strength indicator. So I searched for components in PatternFly and found exactly that in [PFv5](https://www.patternfly.org/components/password-strength/html-demos).

Currently the Admin UI uses PatternFly 4. Are there any plans to upgrade to PFv5 and if so, when can we expect it?

**Comment by ssilvert** [comment]
We plan to upgrade all UIs to PatternFly 5 as soon as we can.  I can't give an exact timeline but I think it will happen relatively fast.

The idea of showing the password rules is a really good one and it has been discussed a lot over the years.  I think we should do it, but we've never found good agreement on how we should do the implementation.   Maybe if someone just submits a good impl then we can push it through.

**Comment by jonkoops** [comment]
See #21321
